wenet/utils/common.py

```diff
@@ -84,6 +111,52 @@ def add_sos_eos(ys_pad: torch.Tensor, sos: int, eos: int,
     return pad_list(ys_in, eos), pad_list(ys_out, ignore_id)
 
 
+def add_eos_sos(ys_pad: torch.Tensor, sos: int, eos: int,
```

Comment:
eos and sos has the same meaning in both forward and backward decoder, there is no need to add the new function.